### PR TITLE
Update protobuf version.

### PR DESCRIPTION
**Pull request by josiah-tesfu**
Update protobuf version and imports after reformatting the edgepi-rpc-protobuf repository.

After this pr is merged I can merge the pr for the rpc server.

### CODE REVIEW

File: edgepi_rpc_client/client_rpc_channel.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import logging
 from google.protobuf import service
 import zmq
-from edgepirpc.protos import rpc_pb2 as rpc_pb
+from python_rpc.protobufs import rpc_pb2 as rpc_pb
```
Comment (farzadpanahi) [inline edgepi_rpc_client/client_rpc_channel.py]: pkg name should not be `python_rpc.protobufs`. it should be `edgepi_rpc_protobuf`

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

**Review by farzadpanahi — DISMISSED** [review]

File: package-lock.json
Comment (farzadpanahi) [inline package-lock.json:1]: what is this file for?

File: package.json
Comment (farzadpanahi) [inline package.json:1]: what is this file for?

File: tests/integration/test_pwm_service.py [hunk 2]
```diff
@@ -119,12 +119,12 @@ def test_get_enabled(pwm_service, pwm_num):
...
+    """Used for manual testing of PWM functionality."""
+    pwm_service.init_pwm(pwm_num=PWMPins.PWM2)
+    pwm_service.set_config(pwm_num=PWMPins.PWM2, polarity=Polarity.NORMAL)
+    pwm_service.enable(PWMPins.PWM2)
     time.sleep(3)
-    pwm_service.set_config(pwm_num=PWMPins.PWM1, duty_cycle=0.5)
+    pwm_service.set_config(pwm_num=PWMPins.PWM2, polarity=Polarity.INVERSED)
     time.sleep(3)
-    pwm_service.set_config(pwm_num=PWMPins.PWM1, duty_cycle=1)
+    pwm_service.set_config(pwm_num=PWMPins.PWM2, polarity=Polarity.NORMAL)
```
Comment (farzadpanahi) [inline tests/integration/test_pwm_service.py:129]: are there any return values from those services that we can assert on?

**Review by farzadpanahi — CHANGES_REQUESTED** [review]
tests are failing!

**Review by naomiven — DISMISSED** [review]
looks good

**Review by farzadpanahi — DISMISSED** [review]

**Review by flavius-t — DISMISSED** [review]

File: .github/workflows/python-integration-test.yml [hunk 3]
```diff
@@ -19,29 +19,52 @@ on:
...
+          deb [signed-by=/usr/share/keyrings/edgepi-release.gpg] https://deb.edgepi.com/main bullseye main
+          deb [signed-by=/usr/share/keyrings/edgepi-release.gpg] https://deb.edgepi.com/test bullseye test
+          deb [signed-by=/usr/share/keyrings/edgepi-release.gpg] https://deb.edgepi.com/nightly bullseye nightly
+          EOF
+
+      - name: Update APT and Install EdgePi RPC Server
+        run: |
+          sudo apt-get update
+          apt-cache pkgnames | grep '^edge'
+          sudo apt-get install edgepi-rpc-server
```
Comment (farzadpanahi) [inline .github/workflows/python-integration-test.yml]: remove these steps from the workflow and add them to the readme.md file under intg test folder. mention that these are the requirements for the intg. tests to run.

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

**Review by naomiven — DISMISSED** [review]

**Review by farzadpanahi — APPROVED** [review]

File: tests/integration/test_pwm_service.py (hunk 2 above)
Comment (josiah-tesfu) [reply tests/integration/test_pwm_service.py:129]: Yes, but but I already did that for all these functions in the previous integration tests. This one was to just manually see on the EdgePi if it worked. For example, I used a fan connected to the EdgePi to see the changes.

**Pull request merged**